src/zh/breaking-changes/v-model.md

```diff
@@ -144,7 +144,7 @@ this.$emit('update:title', newValue)
 <ChildComponent v-model.capitalize="pageTitle" />
 ```
 
-我们可以在[组件事件](http://cn.vuejs.org/guide/components/events.html#usage-with-v-model)部分中了解有关自定义 `v-model` 修饰符的更多信息。
+[自定义 `v-model` 修饰符](https://cn.vuejs.org/guide/components/v-model.html)的更多信息。
```

Comment:
I don't know if this is right.

Reply:
@Jinjiang 
Could you review this fix, please? 🙏

Reply:
It looks great! Thanks.